main.py

```diff
@@ -129,28 +161,21 @@ def loadFile(self):
 
     def loadDirectory(self):
         """Loads whole directory based on Parse class as simple as BHP"""
-        fileDialog = QtWidgets.QFileDialog()
```

Comment:
I don't think this kind of refactoring is necessary.

Reply:
file is a keyword in Python. It should not be used as a variable

Reply:
file - yes, fileDialog - no

Reply:
it's just code reusability